src/ontology/mondo-edit.obo

```diff
@@ -574805,6 +574808,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-12T13:35:21Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7493" xsd:anyURI
 
+[Term]
+id: MONDO:1040012
+name: PI4KA-Related Disorder
+def: "Characterized primarily by neurologic dysfunction (limb spasticity, developmental delay, intellectual disability, seizures, ataxia, nystagmus), gastrointestinal manifestations (multiple intestinal atresia, inflammatory bowel disease), and combined immunodeficiency (leukopenia, variable immunoglobulin defects). Age of onset is typically antenatal or in early childhood; individuals can present with any combination of these features. Rare individuals present with later-onset hereditary spastic paraplegia. Brain MRI findings can include hypomyelinating leukodystrophy, cerebellar hypoplasia/atrophy, thin or dysplastic corpus callosum, and/or perisylvian polymicrogyria." [https://www.clinicalgenome.org/affiliation/40020/, PMID:35951779]
```

Comment:
Definition should start with a "genus" (eg. "a disorder that is characterized....")
I think this term can also follow the "disease by gene" pattern.

Reply:
Comment addressed.